flink-python/dev/dev-requirements.txt

```
@@ -17,7 +17,7 @@ wheel
 apache-beam==2.27.0
 cython==0.29.24
 py4j==0.10.9.3
-python-dateutil==2.8.0
+python-dateutil==2.8.2
```

Keep it consistent with setup.py?
```suggestion
python-dateutil>=2.8.0,<3
```